_data/companies/vonage.yml

```
@@ -1,6 +1,6 @@
 ---
-name: Vonage London
-wfh: Encouraged
+name: Vonage
+wfh: Requires
```

```suggestion
wfh: Required
```
Switching to required for consistency